docs/related.md

```diff
@@ -33,7 +33,7 @@ or [pull request](https://github.com/helm/helm/pulls).
 - [Helm Diff](https://github.com/databus23/helm-diff) - Preview `helm upgrade` as a coloured diff
 - [Helm Value Store](https://github.com/skuid/helm-value-store) - Plugin for working with Helm deployment values
 - [Technosophos's Helm Plugins](https://github.com/technosophos/helm-plugins) - Plugins for GitHub, Keybase, and GPG
-- [helm-convert](https://github.com/ContainerSolutions/helm-convert) - Plugin to convert charts into Kustomize compatible packages
+- [helm-convert](https://github.com/ContainerSolutions/helm-convert) - Plugin to convert charts into Customize compatible packages
```

Comment:
In this case it should be Kustomize as that is the name of a particular app/feature. Can you revert this one.

Reply:
Thank you for your review. I've just reverted it.